languages/en.ini

```diff
@@ -1561,7 +1561,7 @@ You do not have any holds or recalls placed = "You do not have any holds or reca
 You do not have any interlibrary loan requests placed = "You do not have any interlibrary loan requests placed"
 You do not have any items checked out = "You do not have any items checked out"
 You do not have any library cards = "You do not have any library cards"
-You do not have any saved resources = "You do not have any saved resources. Perform a search and use the Save to List button to save items."
+You do not have any saved resources = "You have not saved any favorites. Perform a search and use the Save to List button to save items."
```

Comment:
In #2020, we addressed a long standing inconsistency where some lines referred to "favorites" and others referred to "saved items" in favor of using "saved items" everywhere. I think it would probably be better not to reintroduce that inconsistency in just one line. If you think it would be better to change "resources" to "items" for consistency with other language, I would not object to that adjustment!

(I also wouldn't close the door on a conversation about bringing back "favorites" but I think if we want to use that term, we need to make a lot more changes than just one line).

In any case, thanks for the various improvements here; I just think we should sort out this detail before moving forward with a merge!

Reply:
Thank you for your feedback. I was not aware of this earlier conversation, but had suspicions about "favorites" not being widely used.
I will amend my request. I think "items" instead of "resources" is an acceptable solution.